### PR TITLE
Improve Ukrainian support and minor fixes in CZ, KZ, LT, LV, PL, RU, SR languages

### PR DESCRIPTION
**Pull request by rostyslav80**
### Changes proposed in this pull request:

* Fix English char 'i' in Ukrainian words
* Fix the feminine currency processing for UAH in lang_UK.py
* Fix the feminine currency processing for UAH in lang_RU.py
* Add world currencies to lang_UK.py
* Fix incorrect handling of zeros after decimal point for CZ, KZ, LT, LV, PL, RU, SR and UK languages
* Add Ukrainian ordinal numbers

### Status

- [X] READY
- [ ] HOLD
- [ ] WIP (Work-In-Progress)

### How to verify this change

*Unit tests cover all of the possible cases.*

### Additional notes

*-*

### CODE REVIEW

**Comment by rostyslav80** [comment]
@mromdhane
Could I kindly ask you for a review?

**Comment by rostyslav80** [comment]
Hello, @mromdhane
Any update on this PR?

**Comment by mromdhane** [comment]
Hello, @rostyslav80 can you just give us a little example of bug fixing, please? 
Thank you

**Comment by rostyslav80** [comment]
@mromdhane 
Here is the bug fixing [example](https://github.com/savoirfairelinux/num2words/pull/400/commits/cad28e3f2987cc0b717f53cf0f1595e9324fe36c#diff-056f5faf5dc2e2c425402738777e74a1a96b84902ca698d4de6c38ea4c89561e)

**Before fixing:**
num2words(10.02, lang='cz') -> "deset celá dva"
num2words(15.007, lang='cz') -> "patnáct celá sedm"

**After fixing:**
num2words(10.02, lang='cz') -> "deset celá **nula** dva"
num2words(15.007, lang='cz') -> "patnáct celá **nula nula** sedm"

**Comment by mromdhane** [comment]
@rostyslav80 Thank you!

**Review by mromdhane — APPROVED** [review]

File: num2words/lang_KZ.py [hunk 1]
```diff
@@ -77,10 +77,11 @@ def to_cardinal(self, number):
         n = str(number).replace(',', '.')
         if '.' in n:
             left, right = n.split('.')
+            leading_zero_count = len(right) - len(right.lstrip('0'))
             return u'%s %s %s' % (
                 self._int2word(int(left)),
                 self.pointword,
-                self._int2word(int(right))
+                (ZERO + ' ') * leading_zero_count + self._int2word(int(right))
```
[the remark this answers is not in the document]
Comment (rostyslav80) [reply num2words/lang_KZ.py:84]: @mromdhane This fix is practically the same as for CZ language:

**Before fixing:**
num2words(10.02, lang='kz') -> "он бүтін екі"
num2words(15.007, lang='kz') -> "он бес бүтін жеті"

**After fixing:**
num2words(10.02, lang='kz') -> "он бүтін **нөл** екі"
num2words(15.007, lang='kz') -> "он бес бүтін **нөл нөл** жеті"

**Pull request merged**